### DISCUSSION TITLE
The use of version 1.18.0 of Beanie may have issues.

### DISCUSSION THREAD

**Discussion by hgalytoby** [fastapi-users/fastapi-users#1186] [General]
When using update in Beanie 1.18.0, there is no return value, which causes the program to throw an error.

-----------------------------
api `/users/me`

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L465

updated_user is None

-----------------------------

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L676

self._update

-----------------------------

https://github.com/fastapi-users/fastapi-users-db-beanie/blob/main/fastapi_users_db_beanie/__init__.py#L98

beanie update 

-----------------------------

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L467

return None

-----------------------------

https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/router/users.py#L99

An error occurs because it returns None.

```
pydantic.error_wrappers.ValidationError: 1 validation error for UserRead
email
  field required (type=value_error.missing)
```

I have already raised the issue on the Beanie discussion forum.

https://github.com/roman-right/beanie/discussions/530

**Comment by gegnew** [comment]
I have also noticed this since updating

**Comment by frankie567** [comment]
Shall be fixed with [`fastapi-users-db-beanie==2.0.0`](https://github.com/fastapi-users/fastapi-users-db-beanie/releases/tag/v2.0.0).
